pkg/controller/computenodeopenstack/computenodeopenstack_controller.go

```diff
@@ -588,16 +610,20 @@ func updateNodesStatus(c client.Client, kclient kubernetes.Interface, instance *
 			}
 			// b) run post drain job
 			nodePostDrained, err := ensureNodePostDrain(c, node.Name, instance, openstackClientAdmin, openstackClient)
-			if err != nil {
+			if err != nil && !errors.IsAlreadyExists(err) {
```

Comment:
same here, better to have the if in the opposite order

Reply:
done